### PR TITLE
[DYN-3342] Close stale issues

### PR DESCRIPTION
**Pull request by Astul-Betizagasti**
### Purpose

Add a new workflow that will close issues that have been stale for 30 days or more and have the "need more info" label.
This workflow uses the [Stale](https://github.com/actions/stale) action that is maintained by GitHub.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

Aaron Tang (@QilongTang )

### FYIs

Alfredo Pozo (@alfredo-pozo )

### CODE REVIEW

File: .github/workflows/close_stale_issues.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  close-stale-issues:
+    runs-on: ubuntu-latest
+    steps:
+    - name: close stale issues
+      uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        #This message is requiered but never used.
+        stale-issue-message: "."
```
Comment (QilongTang) [inline .github/workflows/close_stale_issues.yaml:14]: What does this mean?

Comment (Astul-Betizagasti) [reply .github/workflows/close_stale_issues.yaml:14]: By default the action will leave a message when marking the issue as stale and another-one when closing the issue. As we only want the message to be added when the issue is closed the "skip-stale-issue-message" parameter was set to true but the action wont work if the "stale-issue-message" doesn't have a value.

Comment (QilongTang) [reply .github/workflows/close_stale_issues.yaml:14]: @Astul-Betizagasti does empty string works in this case?

Comment (QilongTang) [reply .github/workflows/close_stale_issues.yaml:16]: Thanks for the updates

**Review by QilongTang — APPROVED** [review]
LGTM

**Pull request merged**